app/controllers/phenotype_comments_controller.rb

```diff
@@ -2,45 +2,48 @@ class PhenotypeCommentsController < ApplicationController
   before_filter :require_user
 
   def new
-		@phenotype_comment = PhenotypeComment.new
-		@title = "Add comment"
+    @phenotype_comment = PhenotypeComment.new
+    @title = "Add comment"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.